tests/e2e/configs/sh-scripts/initDefaultValues.sh

```diff
@@ -17,8 +17,8 @@ echo "TS_SELENIUM_VALUE_OPENSHIFT_OAUTH =   ${TS_SELENIUM_VALUE_OPENSHIFT_OAUTH}
 echo "TS_OCP_LOGIN_PAGE_PROVIDER_TITLE =    ${TS_OCP_LOGIN_PAGE_PROVIDER_TITLE}"
 echo "E2E_OCP_CLUSTER_VERSION =             ${E2E_OCP_CLUSTER_VERSION}"
 echo "TS_SELENIUM_LOG_LEVEL =               ${TS_SELENIUM_LOG_LEVEL}"
-echo "TS_SELENIUM_USERNAME =                ${TS_SELENIUM_USERNAME}"
-echo "TS_SELENIUM_PASSWORD =                ${TS_SELENIUM_PASSWORD}"
+echo "TS_SELENIUM_OCP_USERNAME =            ${TS_SELENIUM_OCP_USERNAME}"
+echo "TS_SELENIUM_OCP_PASSWORD =            ${TS_SELENIUM_OCP_PASSWORD}"
```

Comment:
Not sure if it secure to print user password in test log.

Reply:
fixed